Formula/kaitai-struct-compiler.rb

```diff
@@ -0,0 +1,32 @@
+class KaitaiStructCompiler < Formula
+  desc "Compiler for generating binary data parsers"
+  homepage "http://kaitai.io/"
+  url "https://bintray.com/kaitai-io/universal/download_file?file_path=0.7%2Fkaitai-struct-compiler-0.7.zip"
```

Comment:
https://bintray.com/artifact/download/kaitai-io/universal/0.7/kaitai-struct-compiler-0.7.zip is probably a better url

Reply:
waaaay better thank you